superset/databases/commands/exceptions.py

```diff
@@ -176,3 +176,11 @@ class DatabaseOfflineError(SupersetErrorException):
 
 class InvalidParametersError(SupersetErrorsException):
     status = 422
+
+
+class SSHTunnelDeleteFailedError(DeleteFailedError):
+    message = _("SSH Tunnel could not be deleted.")
```

Comment:
can you add a status `500` on this one

Reply:
The parent class already has `status = 500` so it should be already there.